src/sage/rings/ring.pyx

```diff
@@ -1474,32 +1487,18 @@ cdef class Field(CommutativeRing):
 
 
 cdef class Algebra(Ring):
-    """
-    Generic algebra
-    """
-    def __init__(self, base_ring, names=None, normalize=True, category=None):
-        """
-        Initialize ``self``.
```

Comment:
Why remove the docstring?

Reply:
To make it very clear that this is *really deprecated and essentially removed*.